app/modules/price.js

```diff
@@ -0,0 +1,121 @@
+// @flow
+import axios from 'axios'
+import asyncWrap from '../core/asyncHelper'
+import { DEFAULT_CURRENCY_CODE } from '../core/constants'
+
+/* @todo remove below when we upgrade neon-js and can use their getPrice function */
+const CURRENCY = ['aud', 'brl', 'cad', 'chf', 'clp', 'cny', 'czk', 'dkk', 'eur', 'gbp', 'hkd', 'huf', 'idr', 'ils', 'inr', 'jpy', 'krw', 'mxn', 'myr', 'nok', 'nzd', 'php', 'pkr', 'pln', 'rub', 'sek', 'sgd', 'thb', 'try', 'twd', 'zar']
```

Comment:
maybe use Object.keys on the currency from the core constants file instead of creating this list